task/prefetch-dependencies-oci-ta/0.1/prefetch-dependencies-oci-ta.yaml

```diff
@@ -46,11 +50,6 @@ spec:
       description: The name of the ConfigMap to read CA bundle data from.
       type: string
       default: trusted-ca
-    - name: config-file-content
```

Comment:
Why is config file being removed?

Reply:
hm.. its not intentional. It looks like this change just didn't get pulled in via rebase for some reason. Will add it back.

Reply:
should be resolved